basyx/aas/adapter/http.py

```diff
@@ -713,7 +823,7 @@ def post_submodel_submodel_elements_id_short_path(self, request: Request, url_ar
                                                            is_stripped_request(request))
         try:
             parent.add_referable(new_submodel_element)
-        except KeyError:
+        except model.AASConstraintViolation:
```

Comment:
Hmm, this is the correct change, given that an exception in the SDK was changed from `KeyError` to `AASConstraintViolation`. However, we only want to handle a certain kind of `AASConstraintViolation` here, namely the code `22`, which constrains duplicate id_shorts. Maybe we should check the code of the `AASConstraintViolation` to equal `22`, and otherwise re-raise the original exception.